src/permissions/types.ts

```diff
@@ -13,6 +13,11 @@ export interface RepositoryConfig {
   settings?: Partial<RepoSettings>;
   visibility?: 'public' | 'private' | 'current';
   properties?: Record<string, string>;
+  heroku?: {
```

Comment:
Could there be a scenario in which a single repo might be deployed to multiple Heroku apps? If so, I think that could be addressed by making this an array of configs.

Reply:
I don't think so 🤔 Can adjust in the future if required, but for now that's not a scenario we have to consider